src/components/Header/Header.css

```diff
@@ -0,0 +1,99 @@
+@import url('https://fonts.googleapis.com/css2?family=Cairo:wght@300;400&display=swap');
+
+.Container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-top: 3rem;
+}
+
+.Container_1 {
+    color: #fff;
+    display: flex;
+    margin-right: 10rem;
+}
+
+.Container img {
+    margin-right: 2rem;
+}
+
+.Container_1 p {
+    position: relative;
+    font-size: 2rem;
+    font-weight: 600;
+    line-height: 3rem;
+}
+
+.borderBottom {
+    position: absolute;
+    background-color: white;
+    width: 100%;
+    height: 1px;
+    left: 0;
+    bottom: 8px;
+}
+
+.Container_1 p img {
+    position: absolute;
+    top: 10%;
+    right: -32%;
+}
+
+button {
+    background: rgba(0, 0, 0, 0.6);
+    color: #fff;
+    border: 2px solid #5C5C5C;
+    border-radius: 4px;
+    padding: 0.5rem 1rem;
+    background: transparent;
+}
+
+.Container_2 {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+button .StarUs {
+    font-family: 'Cairo', sans-serif;
+    font-size: 1rem;
+    font-weight: 500;
+    margin-right: 1rem;
+}
+
+button .vertical {
+    border-left: 1px solid #5C5C5C;
```

Comment:
```border-left: 2px solid #5C5C5C``` would be great.
Also, add a hover-state to the button which is shown in the Figma design.
![image](https://user-images.githubusercontent.com/86405648/195866240-a9a6793e-fa75-4172-98da-e5aefbce0813.png)